r/tests/testthat/test-backwards-compatibility.R

```diff
@@ -50,7 +50,15 @@ test_that("reading a known Parquet file to dataframe with 3.0.0", {
   pq_file <- test_path("golden-files/data-arrow-extra-meta_3.0.0.parquet")
 
   df <- read_parquet(pq_file)
-  # this is equivalent to `expect_identical()`
+
+  # test the shape (but not the contents of) the column-level attributes for the
+  # large column since those are random and seed changes will make them different
```

Comment:
Why are they random?

Reply:
They don't necessarily have to be, but I was following the pattern in [`make_big_string()`](https://github.com/apache/arrow/blob/master/r/tests/testthat/helper-data.R#L66-L69)